docs/get-started/deployment-and-usage.md

```diff
@@ -319,20 +323,21 @@ of its certificate.
 [cert-manager](https://cert-manager.io/) can be used to automate certificate
 management between nfd-master and the nfd-worker pods. The instructions below describe
 steps how to set up cert-manager's
+First edit the kustomize file under `templates/default` to point to
+the certmanager template base.
 [CA Issuer](https://cert-manager.io/docs/configuration/ca/) to
 sign `Certificate` requests for NFD components in `node-feature-discovery` namespace.
 
 ```bash
-$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
-$ make yamls
-$ openssl genrsa -out ca.key 2048
-$ openssl req -x509 -new -nodes -key ca.key -subj "/CN=nfd-ca" -days 10000 -out ca.crt
-$ sed s"/tls.key:.*/tls.key: $(cat ca.key|base64 -w 0)/" -i nfd-cert-manager.yaml
-$ sed s"/tls.crt:.*/tls.crt: $(cat ca.crt|base64 -w 0)/" -i nfd-cert-manager.yaml
-$ kubectl apply -f nfd-cert-manager.yaml
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
+openssl genrsa -out ca.key 2048
+openssl req -x509 -new -nodes -key ca.key -subj "/CN=nfd-ca" -days 10000 -out ca.crt
+sed s"/tls.key:.*/tls.key: $(cat ca.key|base64 -w 0)/" -i templates/certmanager/nfd-cert-manager.yaml
+sed s"/tls.crt:.*/tls.crt: $(cat ca.crt|base64 -w 0)/" -i templates/certmanager/nfd-cert-manager.yaml
+kubectl apply -k https://raw.githubusercontent.com/kubernetes-sigs/node-feature-discovery/{{ site.release }}/templates/certmanager
 ```
 
-Finally, deploy `nfd-master.yaml` and `nfd-worker-daemonset.yaml` with the Secrets
+This will deploy `nfd-master.yaml` and `nfd-worker-daemonset.yaml` with the Secrets
 (`nfd-master-cert` and `nfd-worker-cert`) mounts enabled.
 
 ## Worker configuration
```

Comment:
with kustomize we could also document how to deploy from NFD repo but with a custom local configuration:

```
bases:
- github.com/ArangoGutierrez/node-feature-discovery/templates/default?ref=devel/issue398

configMapGenerator:
- name: nfd-worker-conf
  namespace: node-feature-discovery
  behavior: replace
  files:
  - nfd-worker-custom.conf
 ```